libs/sdk-ui-kit/src/@ui/UiPagedVirtualList/UiPagedVirtualList.tsx

```diff
@@ -144,6 +146,40 @@ function useVirtualList<T>(props: UiPagedVirtualListProps<T>) {
         }
     }, [hasNextPage, loadNextPage, itemsCount, isLoading, virtualItems, skeletonItemsCount]);
 
+    // Add scroll into view effect
+    useEffect(() => {
+        if (!scrollToItem || !items || !scrollContainerRef.current) {
+            return;
+        }
+
+        const index = items.indexOf(scrollToItem);
```

Comment:
I think this needs to be documented on the relevant prop that provided items must be memoized as object identity is checked inside the component. Alternatively, predicate must be provided as a prop. The second approach is probably more versatile.

Reply:
Updated it.